bft/dialects/types.py

```diff
@@ -161,8 +159,6 @@ def __supports_case_kernel(
             if matched:
                 return None
 
-        if not arg_len_matched:
```

Comment:
Long term we are going to want to be strict about the argument counts.  Perhaps add an issue to track bringing these back?

Reply:
filed issue https://github.com/substrait-io/bft/issues/103 . Thanks!